custom/setup.cfg

```diff
@@ -0,0 +1,35 @@
+[flake8]
```

Comment:
This is a good point. Do we need a separate `setup.cfg`. I guess it's not possible to re-use the values from the top-level one.